docs/pages/bare/hello-world.md

```diff
@@ -1,78 +1,39 @@
 ---
-title: Up and Running
-sidebar_label: Up and Running
+title: Overview
+sidebar_title: Introduction
+hideTOC: true
 ---
 
+import { BoxLink } from '~/ui/components/BoxLink';
 import { Terminal } from '~/ui/components/Snippet';
+import { InlineCode } from '~/components/base/code';
 
-> This guide assumes that you have Xcode and/or Android Studio installed and working. It also assumes that you want to create a new project. If you have an existing app that you would like to integrate the Expo SDK in an existing app, read the [existing apps guide](../bare/existing-apps.md).
+A bare React Native app is a project where developers make direct changes to their native `ios` and `android` project directories, rather than continuously generating them on demand using the [Expo config (**app.json**) and prebuild](/workflow/prebuild). All tools and services offered by Expo including [EAS](/eas), Expo CLI, and the libraries in the Expo SDK are built around bare React Native apps.
```

Comment:
```suggestion
A bare React Native app is a project where developers make direct changes to their native `ios` and `android` project directories rather than continuously generating them on demand using the [Expo config (**app.json**) and prebuild](/workflow/prebuild). All tools and services offered by Expo including [EAS](/eas), Expo CLI, and the libraries in the Expo SDK, are built around bare React Native apps.
```